services/ProductService.js

```diff
@@ -0,0 +1,82 @@
+import axios from 'axios';
+
+const apiUrl = 'https://panda-market-api-crud.vercel.app';
+
+async function getProductList(page, pageSize, keyword) {
+    try {
+        const res = await axios.get(apiUrl+'/products', {
+            page,
+            pageSize,
+            keyword
+        });
+
+        // console.log(res.data);
+        return res.data;
+    } catch (e) {
+        console.error(e);
+    }
+};
+
+async function getProduct(productId) {
+    try {
+        const res = await axios.get(apiUrl+`/products/${productId}`)
+
+        // console.log(res.data);
+        return res.data;
+    } catch (e) {
+        console.error(e);
+    }
+};
+
+async function createProduct(name, description, price, tags, images) {
+    try {
+        const res = await axios.get(apiUrl+'/products', {
```

Comment:
올바른 HTTP 메서드를 사용했는지 확인해주세요 😢

```suggestion
        const res = await axios.post(apiUrl+'/products', {
```